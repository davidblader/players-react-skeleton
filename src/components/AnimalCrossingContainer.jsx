import React from 'react';
import PropTypes from 'prop-types';

const AnimalCrossingContainer = ({ children, ...rest }) => (<div {...rest} className="animal-crossing-container">{children}</div>);

AnimalCrossingContainer.propTypes = {
  children: PropTypes.node.isRequired,
};

export default AnimalCrossingContainer;
