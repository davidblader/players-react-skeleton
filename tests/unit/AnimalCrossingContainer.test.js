import React from 'react';
import { shallow } from 'enzyme';
import AnimalCrossingContainer from '../../src/components/AnimalCrossingContainer';

test('AnimalCrossingContainer shallow renders', () => {
  const component = shallow(<AnimalCrossingContainer>Test</AnimalCrossingContainer>);
  expect(component).toMatchSnapshot();
});
