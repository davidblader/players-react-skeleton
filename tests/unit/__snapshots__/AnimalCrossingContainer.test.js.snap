// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimalCrossingContainer shallow renders 1`] = `
<div
  className="animal-crossing-container"
>
  Test
</div>
`;
